<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_filme_3</class>
 <widget class="QMainWindow" name="tela_filme_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>1101</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 900 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QFrame" name="inicial_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(34, 31, 31);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="btn_Cine_filme_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);
font: 900 22pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>CineFilmes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Recomendacao_aleatorio_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Recomendação aleatório</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Inicio_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 0);
color: rgb(255, 0, 0);
background-color: rgb(34, 31, 31);</string>
      </property>
      <property name="text">
       <string>Inicio</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_Filmes_Series_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 0);
color: rgb(255, 0, 0);
background-color: rgb(34, 31, 31);</string>
      </property>
      <property name="text">
       <string>Filmes e Séries</string>
      </property>
     </widget>
     <widget class="QLabel" name="btn_icon_perfil">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../qrc/filmedois.qrc">:/perfil/icon_perfil.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="img_John_Wicked_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../qrc/filmetres.qrc">:/icon 1/John_Wick_3.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="txt_John_Wick_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>John Wick 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_16_de_Maio_2019">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>16 de Maio de 2019 I 2h 12 minutos I Ação</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Direcao_Chad_Stahel_Roteiro_Derek">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Direção: Chad Stahel I Roteiro: Derek kolstad, Shay Hatten</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Elenco_Keanu_Reeves_Iam_McShanne_Halle_Berry">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>281</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Elenco: Keanu Reeves, Ian McShane,Halle Barry</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_Sinopse_John_Wick_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>371</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Sinopse:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;O lendário John Wick luta para sair de Nova York quando um contrato de 14 milhões de dólares faz dele o alvo dos maiores assassinos do mundo&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="img_Single_Seoul">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>80</y>
       <width>81</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../qrc/filmetres.qrc">:/icon2/Single seul wook.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Single_Seoul">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 9pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Single Seoul</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_29_de_Novembro_de_2023">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>110</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>29 de Novembro de 2023 I 1h 43 Minutos I Romance</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Direcao_Park_Beom_Su">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>130</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Direção: Park Beom-Su I Roteiro: Lee Ji-min</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Elenco_Lee_Dong_Wook">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>150</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Elenco: Lee Dong Wook, Lee Su-Jeong, Esom</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_Single_Seoul_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>220</y>
       <width>371</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Sinopse:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;O filme segue Yeong-ho, que gosta de ficar sozinho, e Hyeon-jin, um editor gregário, que escreve um ensaio sobre a vida de solteiro.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="img_One_Piece_Red">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>71</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../qrc/filmetres.qrc">:/icon3/one piece red.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="img_Ghost_in_the_shell">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>310</y>
       <width>81</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../qrc/filmetres.qrc">:/icon 4/ghost in the shell.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="txt_One_Piece_Red">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>One Piece Red</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_2_de_Novembro_de_2022">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>350</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>2 de Novembro de 2022 I 1h 55 minutos I Aventura</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Direcao_Goro_Tanigushi_Roteiro_Tsutomo_Kuroiwa">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>370</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Direção: Goro Taniguchi I Roteiro: Tsutomo Kuroiwa</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Elenco_Mayumi_Tanaka_Kaori_Nazuka_Ikue_Otani">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>390</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Elenco:  Mayumi Tanaka, Kaori Nazuka, Ikue Ôtani</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_Sinopse_One_Piece_Red">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>440</y>
       <width>371</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Sinopse:&lt;br /&gt;Luffy e sua equipe assistem a um show onde a cantora Uta não é outra senão a filha de Shanks.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Ghost_in_the_shel">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>310</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ghost in the Shell</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_30_de_Marco_2017_">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>340</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>30 de Março de 2017 I 1h 47 minutos I Ação, Ficção Científica</string>
     </property>
    </widget>
    <widget class="QLabel" name="Direcao_Rubert_Sanderson_Roteiro_Jamie_Moss_William_Wheeler">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>360</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Direção: Rupert Sanders I Roteiro:  Jamie Moss, William Wheeler</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Elenco_Scarlett_Johansson_Pilou_Asbaek_Takeshi_Kitano">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>380</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Elenco: Scarlett Johansson,  Pilou Asbæk, Takeshi Kitano</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="txt_Sinopse_One_Piece_Red_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>430</y>
       <width>511</width>
       <height>81</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:900; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ffffff;&quot;&gt;Sinopse:&lt;br /&gt;Em um mundo pós-2029, cérebros se fundem facilmente a computadores e a tecnologia está em todos os lugares. Motoko Kusanagi é uma ciborgue com experiência militar que comanda um esquadrão de elite especializado em combater crimes cibernéticos.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_01">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>520</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>01</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_02">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>520</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>02</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_03">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>520</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>03</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_04">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>520</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>04</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_05">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>520</y>
       <width>31</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>05</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../qrc/filmedois.qrc"/>
  <include location="../qrc/filmetres.qrc"/>
 </resources>
 <connections/>
</ui>
